src/resdk/resolwe.py

```diff
@@ -166,10 +177,13 @@ def _initialize_queries(self):
         """Initialize ResolweQuery's."""
         for resource, query_name in self.resource_query_mapping.items():
             slug_field = self.slug_field_mapping.get(query_name, "slug")
-            query = ResolweQuery(self, resource, slug_field=slug_field)
+            QueryClass = self.resource_query_class.get(resource, ResolweQuery)
+            query = QueryClass(self, resource, slug_field=slug_field)
             if query_name in self.query_filter_mapping:
                 query = query.filter(**self.query_filter_mapping[query_name])
             setattr(self, query_name, query)
+        # Use custon query to reduce the number of queries.
```

Comment:
Typo (custon).

Reply:
Could you please explain, I don;t see any difference between what has been done in the loop above?

Reply:
This is an old code introduced before the `resource_query_class`. It is redundant and has to be removed.